src/views/Users/ViewUser/index.jsx

```diff
@@ -0,0 +1,394 @@
+        />
+      </Grid>
+      <Grid item xs={1} className={classes.gridDelete}>
+        <IconButton onClick={() => handleRoleDelete(role, index)}>
+          <DeleteIcon />
+        </IconButton>
+      </Grid>
+    </Grid>
+  );
+  const renderPermission = (permission, index) => (
+    <Grid container spacing={2} key={index}>
+      <Grid item xs={3}>
+        <AutoCompleteText
+          value={defaultToEmptyString(permission.name)}
+          onValueChange={handlePermissionNameChange(permission, index)}
+          getSuggestions={getSuggestions(allPermissions)}
+          label="Name"
+          required
+          disabled={!permission.metadata.isAdditional}
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={4}>
+        <AutoCompleteText
+          multi
+          disabled={!supportsProductRestriction(permission.name)}
+          selectedItems={permission.options.products}
+          onSelectedItemsChange={handleRestrictionChange(
+            permission,
+            'products'
+          )}
+          onValueChange={handleRestrictionTextChange(permission, 'productText')}
+          value={permission.metadata.productText}
+          getSuggestions={getSuggestions(products)}
+          label="Product Restrictions"
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={4}>
+        <AutoCompleteText
+          multi
+          disabled={!supportsActionRestriction(permission.name)}
+          selectedItems={permission.options.actions}
+          onSelectedItemsChange={handleRestrictionChange(permission, 'actions')}
+          onValueChange={handleRestrictionTextChange(permission, 'actionText')}
+          value={permission.metadata.actionText}
+          getSuggestions={getSuggestions(getSupportedActions(permission.name))}
+          label="Action Restrictions"
+          inputProps={{
+            autoFocus: true,
+          }}
+        />
+      </Grid>
+      <Grid item xs={1} className={classes.gridDelete}>
+        <IconButton onClick={() => handlePermissionDelete(permission, index)}>
+          <DeleteIcon />
+        </IconButton>
+      </Grid>
+    </Grid>
+  );
+
+  return (
+    <Dashboard title="Users">
+      {error && <ErrorPanel error={error} />}
+      {isLoading && <Spinner loading />}
+      {!isLoading && (
+        <Fragment>
+          <form autoComplete="off">
+            <Grid container spacing={2}>
+              <Grid item xs>
+                <TextField
+                  required
+                  disabled={!isNewUser}
+                  onChange={handleUsernameChange}
+                  fullWidth
+                  label="Username"
+                  value={username}
+                />
+              </Grid>
+            </Grid>
+            <br />
+            <br />
+            <br />
+            <Typography variant="h5">Roles</Typography>
+            {roles.map(renderRole)}
```

Comment:
Let's make the spacing between "Roles" and the first row similar to the one in the Required Signoff view.


#### http://localhost:9000/users/balrogadmin

![Screen Shot 2019-07-15 at 11 03 21 AM](https://user-images.githubusercontent.com/3766511/61226393-57635400-a6f0-11e9-93bf-6b0feec19814.png)

#### http://localhost:9000/required-signoffs/Firefox

![Screen Shot 2019-07-15 at 11 03 28 AM](https://user-images.githubusercontent.com/3766511/61226442-6e09ab00-a6f0-11e9-9434-c34ff870abfe.png)

This comment also applies to the "Permissions" header and the row below it. By adding some space on top of the text field, it will also make subsequent text fields have proper spacing similar to the required signoff view.

Reply:
I think I've covered all of this in my upcoming push...